### DISCUSSION TITLE
Mp3Tag only export 1 chapter file

### DISCUSSION THREAD

**Discussion by anddietznutz** [seanap/Plex-Audiobook-Guide#107] [General]
I believe I have copied everything correctly.   When I go to use the "Rename Locate Extra Title" action, it only does 1 files and then the next one says it can export it because it already exists.  I noticed the File name in Mp3tag changes to the same thing for every chapter it try's
![Capture](https://github.com/seanap/Plex-Audiobook-Guide/assets/157053023/8ccbc636-4250-4ada-9704-0d8b19ce8bf6)
 to export.

**Comment by seanap** [comment]
Make sure all the tracks are in order, then select all, hit CTRL+K

Once the track# are set then you can rename everything it will rename it like "Awesome Book - pt01.mp3"

**Reply by anddietznutz** [reply]
Thanks you so much.  I spent almost 5 hours going through and redoing all the setup numerous times.
